test/dotnet-sln-remove.Tests/GivenDotnetSlnRemove.cs

```diff
@@ -191,7 +192,7 @@ public void WhenTooManyArgumentsArePassedItPrintsError()
             var cmd = new DotnetCommand()
                 .ExecuteWithCapturedOutput("sln one.sln two.sln three.sln remove");
             cmd.Should().Fail();
-            cmd.StdErr.Should().BeVisuallyEquivalentTo("Unrecognized command or argument 'two.sln'\r\nUnrecognized command or argument 'three.sln'\r\nYou must specify at least one project to remove.");
+            cmd.StdErr.Should().BeVisuallyEquivalentTo($"Unrecognized command or argument 'two.sln'\r\nUnrecognized command or argument 'three.sln'\r\n{CommonLocalizableStrings.SpecifyAtLeastOneProjectToRemove}");
```

Comment:
https://github.com/dotnet/CliCommandLineParser/issues/75

@jonsequitur committed a fix, that will require some changes in the CLI to consume. I will look into it later today. In the meantime, there are no resources for these that I can use here.